elements/pfe-jump-links/src/pfe-jump-links.js

```diff
@@ -0,0 +1,416 @@
+import PFElement from "../../pfelement/dist/pfelement.js";
+
+class PfeJumpLinks extends PFElement {
+  static get tag() {
+    return "pfe-jump-links";
+  }
+
+  get schemaUrl() {
+    return "pfe-jump-links.json";
+  }
+
+  get templateUrl() {
+    return "pfe-jump-links.html";
+  }
+
+  get styleUrl() {
+    return "pfe-jump-links.scss";
+  }
+
+  // static get events() {
+  //   return {
+  //   };
+  // }
+
+  static get PfeType() {
+    return PFElement.PfeTypes.Content;
+  }
+
+  // static get observedAttributes() {
+  //   return [];
+  // }
+
+  constructor() {
+    super(PfeJumpLinks, { type: PfeJumpLinks.PfeType });
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    // If you need to initialize any attributes, do that here
+  }
+
+  disconnectedCallback() {}
+}
+
+class PfeJumpLinksNav extends PFElement {
+  static get tag() {
+    return "pfe-jump-links-nav";
+  }
+
+  get schemaUrl() {
+    return "pfe-jump-links-nav.json";
+  }
+
+  get templateUrl() {
+    return "pfe-jump-links-nav.html";
+  }
+
+  get styleUrl() {
+    return "pfe-jump-links-nav.scss";
+  }
+
+  // static get events() {
+  //   return {
+  //   };
+  // }
+
+  // Declare the type of this component
+  static get PfeType() {
+    return PFElement.PfeTypes.Content;
+  }
+
+  // static get observedAttributes() {
+  //   return [];
+  // }
+
+  constructor() {
+    super(PfeJumpLinksNav, { type: PfeJumpLinksNav.PfeType });
+
+    this._buildNav = this._buildNav.bind(this);
+    this._mutationCallback = this._mutationCallback.bind(this);
+    this._menuContainer = this.shadowRoot.querySelector("#container");
+    this._observer = new MutationObserver(this._mutationCallback);
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    // If you need to initialize any attributes, do that here
+
+    //Check that the light DOM is there
+    if (this.hasAttribute("autobuild")) {
+      this._buildNav();
+    } else {
+      //Check that the light DOM is valid
+      if (this._isValidLightDom()) {
+        const menu = this.querySelector("ul");
+        this._menuContainer.innerHTML = menu.outerHTML;
+      }
+    }
+
+    this._observer.observe(this, {
+      childList: true,
+      subtree: true,
+      characterData: true,
```

Comment:
@Djfaucette - Just curious, why is this component watching for character data changes?

Reply:
In the auto built version, the innerhtml of the element is used to make the innerhtml of its corresponding nav item. Figured since we're watching for that in the panel we should here too?